src/components/ui/image.jsx

```diff
@@ -30,6 +30,15 @@ let Image = React.createClass({
     }
   },
 
+  componentWillReceiveProps(nextProps) {
+    let { src } = this.state
+
+    this.setState({
+      isLoaded: nextProps.src !== src,
+      src: nextProps.src
+    });
+  },
```

Comment:
Ya, what was the bug?

-- 
Nate Hunzaker

On October 19, 2015 at 3:06:22 PM, Chris Manning (notifications@github.com) wrote:

> +    let { src } = this.state